### DISCUSSION TITLE
Feature Request: Ability to Expand All or Collapse All links in a Better Note

### DISCUSSION THREAD

**Discussion by DonnaCoxBaker** [windingwind/zotero-better-notes#1010] [Ideas]
I love being able to display the contents of a note within the Better Note--and to collapse it back out of sight. Would it be possible to press a button and have every link in the Better Note expand (drawing in the outbound link's text), and to press it again to collapse all of them back to links?

**Comment by windingwind** [comment]
I'm against the idea of *expanding the linked note since it's actually *embedding the note. The embedded note content will not be updated if the note is modified.

I'm working on a new feature that allows hovering to preview the linked note (#1004). If you want to expand all note links anyway, you can use Export -> To New Zotero Note -> All Embedded in One Export, which will duplicate the current note with all links embedded.

**Reply by DonnaCoxBaker** [reply]
That sounds like a good alternative. Thanks!

**Reply by windingwind** [reply]
Available in the latest beta (89).

**Comment by DonnaCoxBaker** [comment]
Thanks so much. The hover view is *very* helpful!

                   Visit me at Genohistory.com <https://genohistory.com>
                   Sign up for my Genohistory Notes newsletter!
<https://landing.mailerlite.com/webforms/landing/f0x2k7>
